hyperspy/_signals/image.py

```diff
@@ -40,3 +40,98 @@ def to_spectrum(self):
+            provided to explore the data.
+            If navigation_dimension is 1 and the signal is an image
+            the navigator is a spectrum obtained by integrating
+            over the signal axes (the image).
+            If navigation_dimension is 1 and the signal is a spectrum
+            the navigator is an image obtained by stacking horizontally
+            all the spectra in the dataset.
+            If navigation_dimension is > 1, the navigator is an image
+            obtained by integrating the data over the signal axes.
+            Additionaly, if navigation_dimension > 2 a window
+            with one slider per axis is raised to navigate the data.
+            For example,
+            if the dataset consists of 3 navigation axes X, Y, Z and one
+            signal axis, E, the default navigator will be an image
+            obtained by integrating the data over E at the current Z
+            index and a window with sliders for the X, Y and Z axes
+            will be raised. Notice that changing the Z-axis index
+            changes the navigator in this case.
+            If "slider" and the navigation dimension > 0 a window
+            with one slider per axis is raised to navigate the data.
+            If "spectrum" and navigation_dimension > 0 the navigator
+            is always a spectrum obtained by integrating the data
+            over all other axes.
+            If None, no navigator will be provided.
+            Alternatively a Signal instance can be provided. The signal
+            dimension must be 1 (for a spectrum navigator) or 2 (for a
+            image navigator) and navigation_shape must be 0 (for a static
+            navigator) or navigation_shape + signal_shape must be equal
+            to the navigator_shape of the current object (for a dynamic
+            navigator).
+            If the signal dtype is RGB or RGBA this parameters has no
+            effect and is always "slider".
+        axes_manager : {None, axes_manager}
+            If None `axes_manager` is used.
+        colorbar : bool, optional
+             If true, a colorbar is plotted for non-RGB images.
+        scalebar : bool, optional
+            If True and the units and scale of the x and y axes are the same a
+            scale bar is plotted.
+        scalebar_color : str, optional
+            A valid MPL color string; will be used as the scalebar color.
+        axes_ticks : {None, bool}, optional
+            If True, plot the axes ticks. If None axes_ticks are only
+            plotted when the scale bar is not plotted. If False the axes ticks
+            are never plotted.
+        auto_contrast : bool, optional
+            If True, the contrast is stretched for each image using the
+            percentile value.
+        percentile : float
+            The percentile to be used for contrast stretching. It should be a
+            scalar in the 0 to 1 range.
+        vmin, vmax : scalar, optional
```

Comment:
When I was testing this, it wasn't immediately clear to me why `vmin` and `vmax` didn't work (which was because `auto_contrast` defaults to `True`). Perhaps if these options are supplied, `auto_contrast` should be automatically shut off, or a warning printed to the user that the options conflict? I think this would improve the usability.